tasks/winbuildscripts/integrationtests.ps1

```diff
@@ -0,0 +1,33 @@
+$ErrorActionPreference = "Stop"
+$Password = ConvertTo-SecureString "dummyPW_:-gch6Rejae9" -AsPlainText -Force
+New-LocalUser -Name "ddagentuser" -Description "Test user for the secrets feature on windows." -Password $Password
+
+$Env:Python2_ROOT_DIR=$Env:TEST_EMBEDDED_PY2
+$Env:Python3_ROOT_DIR=$Env:TEST_EMBEDDED_PY3
+
+if ($Env:TARGET_ARCH -eq "x64") {
+    & ridk enable
+}
+& $Env:Python3_ROOT_DIR\python.exe -m  pip install -r requirements.txt
+
+$UT_BUILD_ROOT=(Get-Location).Path
+$Env:PATH="$UT_BUILD_ROOT\dev\lib;$Env:GOPATH\bin;$Env:Python3_ROOT_DIR;$Env:Python3_ROOT_DIR\Scripts;$Env:PATH"
+
+& $Env:Python3_ROOT_DIR\python.exe -m pip install PyYAML==5.3.1
```

Comment:
```suggestion
& $Env:Python3_ROOT_DIR\python.exe -m pip install PyYAML==5.4.1
```
PyYAML is [pinned](https://github.com/DataDog/datadog-agent-buildimages/blob/main/setup_python.sh#L77) at 5.4.1 in buildimages, not sure it makes a great difference but should we align the versions?

Reply:
I co[pied most of that script from the other windows build scripts.

https://github.com/DataDog/datadog-agent/pull/12300 updated it last. 

Your link is for linux, right? I don't see a similar line in the windows build image scripts. I'd expect reno to install the version of PyYAML it needs when it is installed though.

I can make a card to look into fixing it for all the build scripts but I'd like to keep them consistent for this PR.